record/record-writeAcknowledgement.feature

```diff
@@ -30,25 +30,33 @@ Scenario: Record
 	When the client sets the record "happyRecord" "pets.0.name" to "Max"
 	Then the last message the server recieved is R|P|happyRecord|101|pets.0.name|SMax|{"writeSuccess":true}+
 
-	Then the server sends the message R|WS|happyRecord|101|L+
+	Then the server sends the message R|WS|happyRecord|O[101]|L+
```

Comment:
you don't need to use the object encoding O since this will always be an array of numbers